.gitlab-ci.yml

```diff
@@ -201,7 +209,7 @@ trigger_build_kernels:
   extends: .test
   before_script:
     - rm -r /go/src/github.com/DataDog/datadog-agent || true
-    - git clone https://github.com/DataDog/datadog-agent /go/src/github.com/DataDog/datadog-agent
+    - git clone -b $AGENT_TESTING_BRANCH https://github.com/DataDog/datadog-agent /go/src/github.com/DataDog/datadog-agent
```

Comment:
I'm guessing this will be removed after the corresponding agent PR is merged?

Reply:
This will be kept but the variable at the top will be set to `main`